packages/common/src/types/job.ts

```diff
@@ -0,0 +1,9 @@
+export type Job = {
+    jobUuid: string;
+    projectUuid: string;
+    createdAt: Date;
+    updatedAt: Date;
+    jobStatus: string;
```

Comment:
Could we make this an enum / literal type so we know the exact valid value? Just like the database type